js/main.js

```diff
@@ -1,2 +1,74 @@
 $(function () {
+  let playerScore = 0
+  let botScore = 0
+  let playerChoice
+  let botChoice
+
+  $("button").not('#reset').click((event) => {
+    playerChoice =  $(event.currentTarget).attr('id')
+    botChoice = generateBotChoice()
+    compare(botChoice, playerChoice)
+  })
+
+  $('#reset').click(() => {
+     playerScore = 0
+     botScore = 0
+     clearStatus()
+     $('#humanScore').html(playerScore)
+     $('#computerScore').html(botScore)
+   })
+
+  function generateBotChoice() {
+    let botChoice
+    const number = Math.floor(Math.random() * 3) + 1;
+    if(number === 1){
+     botChoice = 'rock'
+      } else if(number === 2){
+        botChoice = 'paper'
+      } else {
+        botChoice = 'scissors'
+      }
+    return botChoice
+  }
+
+  function compare(botChoice, playerChoice) {
+    if(botChoice === playerChoice){
+      clearStatus()
+      $('#status').append(`You played <u>${playerChoice}</u>. The bot played <u>${botChoice}</u>. It's a tie!`)
+    } else if(botChoice === "rock"){
+        if(playerChoice === 'scissors'){
+          clearStatus()
+          $('#status').append(`You played <u>${playerChoice}</u>. The bot played <u>${botChoice}</u>. You Lose :(`)
+          $('#computerScore').html(botScore += 1)
```

Comment:
@nmikhol the compare function does the job, but this code snippet could be extended into another function that updates the status with the respective score and choices passed as arguments. 

Check out [this](https://github.com/jsr20190826/rock-paper-scissors-solution/blob/1583be48146e2521b25308e017c52fbfcf0d9d89/js/main.js#L82) approach